pkg/gce-pd-csi-driver/node.go

```diff
@@ -456,14 +472,18 @@ func (ns *GCENodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUns
 	}
 
```

Comment:
I'd put the `EnableDeviceInUseCheck` check here, and wrap everything. That way you don't have references to confirmDevice or deviceInUse outside of that check (eg: this calls `ns.deviceInUseErrors.deleteDevice` even when the flag is false).

Reply:
Sure, done.